lib/resources/static.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v3 "github.com/projectcalico/libcalico-go/lib/apis/v3"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+)
+
+const (
+	// AllowProfileName is the name of an "allow-all" profile.
+	AllowProfileName = "projectcalico-allow-all"
+)
+
+// AllowProfile returns a single profile kvp with default allow rules.
+// This profile can be attached to host endpoints to allow traffic in the
+// absence of policy.
+func AllowProfile() *model.KVPair {
```

Comment:
This looks like a duplicate file? Do we need both this and the `lib/backend/resources` one?

Reply:
That was an error. I removed it.